OpenTabletDriver.Configurations/Parsers/RobotPen/RobotPenReportParser.cs

```diff
@@ -0,0 +1,15 @@
+using OpenTabletDriver.Plugin.Tablet;
+
+namespace OpenTabletDriver.Configurations.Parsers.RobotPen
+{
+    public class RobotPenReportParser : IReportParser<IDeviceReport>
+    {
+        public IDeviceReport Parse(byte[] report)
+        {
+            if (report[1].ToString() == "66")
+                return new RobotPenTabletReport(report);
+
+            return new DeviceReport(report);
+        }
```

Comment:
Do *not* use strings to compare numerics ever again.

I'm assuming that non-`0x42` reports are out-of-range. Please do correct me if I'm wrong.

```suggestion
        {
            if (report[1] == 0x42)
                return new RobotPenTabletReport(report);

            return new OutOfRangeReport(report);
        }
```